doc/source/ray-core/scheduling/index.rst

```diff
@@ -37,8 +37,13 @@ Currently the supported strategies are the followings.
 "DEFAULT"
 ~~~~~~~~~
 
-``"DEFAULT"`` is the default strategy used by Ray. With the current implementation, Ray will try to pack tasks or actors on nodes
-until the resource utilization is beyond a certain threshold and spread them afterwards.
```

Comment:
Can we keep some high-level description of the policy? The new version of the text is very low-level and it's a bit hard to understand the high-level behavior, which is what most users will care about.